test/routing/RouterProgress.test.tsx

```diff
@@ -66,14 +66,17 @@ describe.skip('Routing - Router Progress', () => {
     await waitFor(() => {
       expect(container.querySelector('h2.vads-u-font-size--h4')).toBeNull();
     });
-    userEvent.click(container.querySelector('button.next')!);
+    userEvent.click(container.querySelector('va-button[submit][continue]')!);
 
     await waitFor(() =>
       expect(
         container.querySelector('h2.vads-u-font-size--h4')?.innerHTML
       ).toContain('Step 1 of 1: About')
     );
-    userEvent.click(container.querySelector('button.next')!);
+    fireEvent(
+      container.querySelector('va-button-pair')!,
+      new CustomEvent('primaryClick')
+    );
```

Comment:
`<va-button-pair>` fires `primaryClick` when the primary button is clicked, so I'm simulating that since the primary button is inside of the shadow DOM and not easily accessed with RTL & JSDOM.

![Screenshot of the `<va-button-pair>` component's Storybook page, with the `primaryClick` event highlighted](https://user-images.githubusercontent.com/2008881/182912860-a5cc5b92-adb8-4e4b-8206-ccf9e0287f0c.png)

Reply:
I pushed a fix for the Page component to enable, (Continue only, Back only, and Continue/Back Pair) which fixed the Page.test.tsx but still no dice on the RouterProgress test